site/en/docs/privacy-sandbox/attribution-reporting/constraints/index.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Constraints on Aggregation Reporting data'
+subhead: >
+  Understand the data constraints, requirements for destinations and origins, and the impact on schedules.
+description: >
+  Understand the data constraints, requirements for destinations and origins, and the impact on schedules.
+date: 2022-03-31
+updated: 2023-07-21
+authors:
+  - maudn
+---
+
+To plan your Attribution Reporting strategy, you'll need to know the constraints enforced by the API, including how much data can be collected for each report.
+
+## Event-level reports
+
+The following sections on click and view events list features and limitations to keep in mind.
+
+### Click (`navigation` source)
+
+- `source_event_id` (ad-side information) can be up to 64 bits - large enough to be be used as a unique identifier.
+- `trigger_data` (conversion-side information) is up to 3 bits; that is, it should be an integer between 0 and 7.
+- Up to 3 conversions can be attributed to a given source (ad click or view event).
+- Noise is applied to reports in the form of a randomized response. The correct conversion count can be recovered via a [script](https://github.com/WICG/attribution-reporting-api/blob/main/noise_corrector.py).
+- Noise is applied with a 0.0024 (0.24%) probability in the current Chrome code.
+- Reports are sent with a delay of between 2 and 30 days; the exact mechanism for view and clicks vary. See details in [Report scheduling for event-level reports](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#heading=h.c5k14yezmbvl).
+
+### View (`event` source)
+
+- `source_event_id` (ad-side information) can be up to 64 bits - large enough to be be used as a unique identifier.
+- `trigger_data` (conversion-side information) is up to 3 bits; that is, it should be either 0 or 1.
+- Up to 1 conversion can be attributed to a given source (ad click or view event).
+- Noise is applied to reports in the form of a randomized response. The correct conversion count can be recovered via a [script](https://github.com/WICG/attribution-reporting-api/blob/main/noise_corrector.py).
+- Noise is applied with a 0.0000025 probability (0.00025%) in the current Chrome code.
+- Reports are sent with a delay of between 2 and 30 days; the exact mechanism for view and clicks vary. See details [Report scheduling for event-level reports](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#heading=h.c5k14yezmbvl).
+
+
+## Aggregatable reports
+
+The following list notes limitations to keep in mind.
+
+### Click (`navigation` source) and View (`event` source)
+
+- The total values of all the buckets attributed to a source should be at most 65536.
```

Comment:
This first bullet and the nested one below look a bit awkward when rendered. I think this could all just be standard prose and then keep the three bullet points at the end.

Reply:
Done thanks!